Sources/SwiftyRequest/RestRequest.swift

```diff
@@ -68,7 +74,7 @@ public class RestRequest: NSObject  {
                                                 maxFailures: params.maxFailures,
                                                 rollingWindow: params.rollingWindow,
                                                 bulkhead: params.bulkhead,
-                                                command: handleInvocation,
+                                                command: { [weak self] param in self?.handleInvocation(invocation: param) },
```

Comment:
Please can you add a comment that explains what we are doing here?  (ie. describes the reference cycle that is being avoided)